src/cudnn/custom_kernels/f_kernels.cu

```diff
@@ -0,0 +1,55 @@
+// every function must end with `f32` and only accept `float`s; then the same function with `f64` and `double`s will be generated
+
+extern "C" __global__ void recip_with_scale_f32(float *out, const float *a, const float *a_scale, size_t numel)

```

Comment:
whoa these are awesome! especially that compile_custom_kernels.sh file. :rocket:

Reply:
thanks :)
(though this doesn’t work on windows for me lol)